modules/civiremote_event/src/Utils.php

```diff
@@ -42,14 +42,23 @@ public static function fieldType($field, $step = 'form') {
     if (isset($field['validation']) && $field['validation'] == 'Email') {
       $type = 'email';
     }
-    // Use radio buttons for select fields with up to 10 options.
+    // Use checkboxes for multi-select fields with up to 10 options.
     elseif (
       $types[$field['type']] == 'select'
       && count($field['options']) <= 10
       // Checkboxes #options may not have a 0 key.
       && !isset($field['options'][0])
+      && $field['type'] == 'Multi-Select'
```

Comment:
Doesn't make this the check `$types[$field['type']] == 'select'` obsolete?

Reply:
Sort of, you're right, but I think it's more obvious that we're explicitly replacing `select` type fields here. At least without it, the general mapping of CiviCRM field types to Drupal ones would not be taken into account anymore.